cloudferrylib/os/identity/keystone.py

```diff
@@ -213,6 +214,8 @@ def deploy(self, info):
         self._upload_user_tenant_roles(roles, users, tenants)
         LOG.info("Done")
 
+    @retrying.retry(expected_exceptions=[ks_exceptions.Unauthorized],
+                    reraise_original_exception=True, max_attempts=5)
```

Comment:
Unfortunately, now we don't have a possibility of using the global `CONF` variable. Let's move this logic inside the function.